### PR TITLE
enh(mc): Great Reboot - network-hp-standard-snmp

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 1]
```diff
@@ -2,36 +2,835 @@
...
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                      | Modèle de service                                      | Description                                                                                   | Découverte |
+|:---------------------------|:-------------------------------------------------------|:----------------------------------------------------------------------------------------------|:----------:|
+| Cast-Generic-Name          | Net-HP-Standard-Cast-Generic-Name-SNMP-custom          | Contrôle le pourcentage de paquets pour chaque type de casting d'une interface réseau         |            |
+| Cast-Global                | Net-HP-Standard-Cast-Global-SNMP-custom                | Contrôle le pourcentage de paquets pour chaque type de casting de plusieurs interfaces réseau | X          |
+| Packet-Errors-Generic-Name | Net-HP-Standard-Packet-Errors-Generic-Name-SNMP-custom | Contrôle le pourcentage de paquets en erreur/écarté d'une interface réseau                    |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:38]:
```suggestion
| Packet-Errors-Generic-Name | Net-HP-Standard-Packet-Errors-Generic-Name-SNMP-custom | Contrôle le pourcentage de paquets en erreur/écartés d'une interface réseau                    |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 2]
```diff
@@ -2,36 +2,835 @@
...
+| Packet-Errors-Global       | Net-HP-Standard-Packet-Errors-Global-SNMP-custom       | Contrôle le pourcentage de paquets en erreur/écarté de plusieurs interfaces réseau            | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:39]:
```suggestion
| Packet-Errors-Global       | Net-HP-Standard-Packet-Errors-Global-SNMP-custom       | Contrôle le pourcentage de paquets en erreur/écartés de plusieurs interfaces réseau            | X          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 3]
```diff
@@ -2,36 +2,835 @@
...
+
+#### Découverte d'hôtes
+
+| Nom de la règle | Description                                                                                                                                                      |
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de service
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:62]:
```suggestion
#### Découverte de services
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 4]
```diff
@@ -2,36 +2,835 @@
...
+
+| Nom de la règle                         | Description                                                           |
+|:----------------------------------------|:----------------------------------------------------------------------|
+| Net-Hp-Standard-SNMP-Cast-Name          |                                                                       |
+| Net-Hp-Standard-SNMP-Packet-Errors-Name | Discover network interfaces and monitor errored and discarded packets |
+| Net-Hp-Standard-SNMP-Spanning-Tree-Name |                                                                       |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:68]: description?

File: pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 5]
```diff
@@ -2,36 +2,834 @@
...
+|:---------------------------|:-------------------------------------------------------|:----------------------------------------------------------------|:----------:|
+| Cast-Generic-Name          | Net-HP-Standard-Cast-Generic-Name-SNMP-custom          | Check packets casting type of a network interface               |            |
+| Cast-Global                | Net-HP-Standard-Cast-Global-SNMP-custom                | Check packets casting type of multiple network interfaces       | X          |
+| Packet-Errors-Generic-Name | Net-HP-Standard-Packet-Errors-Generic-Name-SNMP-custom | Check packets on errors/discards of a network interface         |            |
+| Packet-Errors-Global       | Net-HP-Standard-Packet-Errors-Global-SNMP-custom       | Check packets on errors/discards of multiple network interfaces | X          |
+| Spanning-Tree              | Net-HP-Standard-SpanningTree-SNMP-custom               | Check Spanning Tree state on interfaces                         | X          |
+| Traffic-Generic-Id         | Net-HP-Standard-Traffic-Generic-Id-SNMP-custom         | Check traffic of an network interface                           |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:40]:
```suggestion
| Traffic-Generic-Id         | Net-HP-Standard-Traffic-Generic-Id-SNMP-custom         | Check traffic of a network interface                           |            |
```

File: pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 6]
```diff
@@ -2,36 +2,834 @@
...
+| Traffic-Generic-Name       | Net-HP-Standard-Traffic-Generic-Name-SNMP-custom       | Check traffic of an network interface                           |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:41]:
```suggestion
| Traffic-Generic-Name       | Net-HP-Standard-Traffic-Generic-Name-SNMP-custom       | Check traffic of a network interface                           |            |
```

File: pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 7]
```diff
@@ -2,36 +2,834 @@
...
+
+More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).
+
+#### Service discovery
+
+| Rule name                               | Description                                                           |
+|:----------------------------------------|:----------------------------------------------------------------------|
+| Net-Hp-Standard-SNMP-Cast-Name          |                                                                       |
+| Net-Hp-Standard-SNMP-Packet-Errors-Name | Discover network interfaces and monitor errored and discarded packets |
+| Net-Hp-Standard-SNMP-Spanning-Tree-Name |                                                                       |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:67]: description?

File: pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 8]
```diff
@@ -2,36 +2,834 @@
...
-### SNMP
+</TabItem>
+</Tabs>
+
+## Using the monitoring connector
+
+### Using a host template provided by the connector
+
+1. Log into Centreon and add a new host through **Configuration > Hosts**.
+2. Fill the **Name**, **Alias** & **IP Address/DNS** fields according to your ressource settings.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:264]:
```suggestion
2. Fill in the **Name**, **Alias** & **IP Address/DNS** fields according to your resource's settings.
```

**Review by cg-tw — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md [hunk 9]
```diff
@@ -2,36 +2,835 @@
...
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle                         | Description                                                           |
+|:----------------------------------------|:----------------------------------------------------------------------|
+| Net-Hp-Standard-SNMP-Cast-Name          |                                                                       |
+| Net-Hp-Standard-SNMP-Packet-Errors-Name | Discover network interfaces and monitor errored and discarded packets |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-hp-standard-snmp.md:67]:
```suggestion
| Net-Hp-Standard-SNMP-Packet-Errors-Name | Découvre les interfaces en vue de contrôler les paquets en erreur. |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**